heron/spi/src/java/com/twitter/heron/spi/utils/ShellUtils.java

```diff
@@ -111,11 +112,24 @@ public static Process runASyncProcess(
       boolean verbose, String[] command, File workingDirectory) {
     // Log the command for debugging
     LOG.log(Level.FINE, "$> {0}", Arrays.toString(command));
+    // Log the working directory, this will make people fast locate the
+    // directory to find the log files
+    LOG.log(Level.INFO, "working dir: {0}", workingDirectory.getAbsolutePath());
```

Comment:
@maosongfu Actually, this log is for people who use YARN scheduler, quickly and easily to locate the working directory. For now, only the yarn working directory is too difficult to find in executor machine. So I think this code can move to another class `HeronExecutorTask` which is only for yarn scheduler.
On the other hand, is there a way to easily adjust the log level. If not, then the log level is set to `Level.INFO` will be better.